haas/api.py

```diff
@@ -605,20 +702,29 @@ def show_network(network):
 
     network = _must_find(model.Network, network)
 
-    if network.access is not None:
-        auth_backend.require_project_access(network.access)
+    if network.access:
+        #auth tests: admin owned networks with a project access
+        #TODO only network owner allowed to show the network?
```

Comment:
The thing that bugs me is that this is inconsistent with the decisions we've made on some of the other calls; we don't allow non-owners to see what nodes other projects have on the network, but this allows them to see what projects are there. My inclination is to require ownership of the network for this, but I'd like to get other dev's opinions. @henn, @gsilvis?

Once we come to a decision (either way), the comment should be removed.